transports/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -32,51 +31,36 @@ A typical implementation uses a single storage account to send and receive messa
 
 ![Single storage account](azure01.png "width=500")
 
-When the number of instances with endpoints are increased, all endpoints continue reading and writing to the same storage account. Once the limit of 2,000 message/sec per queue or 20,000 message/sec per storage account is reached, Azure throttles the message throughput.
+When the number of instances of endpoints are increased, all endpoints continue reading and writing to the same storage account. Once the limit of 2,000 message/sec per queue or 20,000 message/sec per storage account is reached, Azure throttles the message throughput.
```

Comment:
> When the number of instances of endpoints are increased

is confusing. How about "When the number of endpoint instances is increased,..."?

Reply:
> Azure throttles the message throughput.

"Storage service throttles..."

Reply:
addressed